### PR TITLE
modify AudioAttachment function

### PR DESCRIPTION
**Pull request by sdnian**
Get audio message information for fetchThreadMessages()

### CODE REVIEW

File: fbchat/models.py [hunk 1]
```diff
@@ -267,10 +267,20 @@ def __init__(self, url=None, size=None, name=None, is_malicious=None, **kwargs):
         self.name = name
         self.is_malicious = is_malicious
 
-class AudioAttachment(FileAttachment):
-    def __init__(self, **kwargs):
-        """Represents an audio file that has been sent as a Facebook attachment - *Currently Incomplete!*"""
-        super(StickerAttachment, self).__init__(**kwargs)
+class AudioAttachment(Attachment):
+    # Name of the file
```
Comment (madsmtm) [inline fbchat/models.py]: These comments should start with a colon, like this: `#: comment`, so they'll show up in the sphinx autogenerated docs on [ReadTheDocs](http://fbchat.readthedocs.io/)

File: fbchat/graphql.py [hunk 2]
```diff
@@ -109,6 +109,12 @@ def graphql_to_attachment(a):
             large_image=a.get('large_image'),
             uid=a.get('legacy_attachment_id')
         )
+    elif _type == 'MessageAudio':
+        return AudioAttachment(
```
Comment (madsmtm) [inline fbchat/graphql.py]: The `a` dict contains an `audio_type`, maybe we should include this in the `AudioAttachment` too?

File: fbchat/models.py [hunk 3]
```diff
@@ -267,10 +267,20 @@ def __init__(self, url=None, size=None, name=None, is_malicious=None, **kwargs):
...
+    filename = None
+    # playable_url
+    playable_url = None
+    # duration
```
Comment (madsmtm) [inline fbchat/models.py]: Maybe you could elaborate on the documentation comments? Something like `Duration of the audioclip in milliseconds`

File: fbchat/models.py [hunk 4]
```diff
@@ -267,10 +267,20 @@ def __init__(self, url=None, size=None, name=None, is_malicious=None, **kwargs):
...
 
-class AudioAttachment(FileAttachment):
-    def __init__(self, **kwargs):
-        """Represents an audio file that has been sent as a Facebook attachment - *Currently Incomplete!*"""
-        super(StickerAttachment, self).__init__(**kwargs)
+class AudioAttachment(Attachment):
+    # Name of the file
+    filename = None
+    # playable_url
+    playable_url = None
```
Comment (madsmtm) [inline fbchat/models.py]: Just using `url` should be sufficiently describing here ;)

**Comment by sdnian** [comment]
Thank you for your reply. I've follow your suggestion to modify these codes. But I don't know how to update? Sorry, this is my first time to do this. And I am not a programmer.

**Comment by madsmtm** [comment]
Think you updated it properly, it looks good now!

**Pull request merged**